src/components/FileUploader/FileUploader-story.js

```diff
@@ -22,16 +23,28 @@ storiesOf('FileUploader', module)
   .addWithInfo(
     'FileUploader',
     `
-      The FileUploader components allow the user to upload any necessary files. This uses the FileUploaderButton and Filename components. Filename components will appear below the FileUploaderButton when files are added. Use the filenameStatus prop to control what icon appears in Filename ('edit', 'complete', or 'uploading').
+      The FileUploader components allow the user to upload any necessary files. This uses the FileUploaderButton and Filename components. Filename components will appear below the FileUploaderButton when files are added. Use the filenameStatus prop to control what icon appears in Filename ('edit', 'complete', or 'uploading'). The FileUploader component contains a method to clear all files uploaded, clearFiles. This can be used with a ref in the parent component. The clear file button in this example is an example of how to use the clearFiles method.
     `,
     () => (
-      <FileUploader
-        labelTitle="Upload"
-        labelDescription="only .jpg and .png files at 500mb or less"
-        buttonLabel="Add files"
-        filenameStatus="edit"
-        accept={['.jpg', '.png']}
-        multiple
-      />
+      <div>
```

Comment:
If we have to have a wrapping div, could we add a classname in case a user needs to target the container specifically? Maybe something like `bx--file__container`

Reply:
Absolutely! I wasn't sure if it mattered for the story.